file-attribute-caching/src/main/kotlin/com/pkware/filesystem/attributecaching/AttributeCachingPath.kt

```diff
@@ -256,44 +277,53 @@ internal class AttributeCachingPath(
         // remove basic: from our attribute name if present as basicFileAttributes can be accessed without that qualifier
         var checkedNames = names.substringAfter("basic:")
 
-        // get our attribute class from the cache, should be BasicFileAttributes, DosFileAttributes, or PosixFileAttributes
-        val attributeClass = if (checkedNames.startsWith("dos")) {
-            dosAttributes
-        } else if (checkedNames.startsWith("posix")) {
-            posixAttributes
+        if (checkedNames.startsWith("acl")) {
```

Comment:
This could just be part of the same if/else ladder that existed before

Reply:
it cannot because we are working with the acl file attribute view vs an object that extends BasicFileAttributes